pages/product/[productAppExtension]/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Panel, Text } from "@bigcommerce/big-design";
+import { useRouter } from "next/router";
+import ErrorMessage from "@components/error";
+import Loading from "@components/loading";
+import { useProductInfo, useProductList } from "@lib/hooks";
+
+const ProductAppExtension = () => {
+    const router = useRouter();
+    const productAppExtension = Number(router.query?.productAppExtension);
+    const { error, isLoading, list = [] } = useProductList();
+    const { isLoading: isInfoLoading, product } = useProductInfo(productAppExtension, list);
+    const { description, is_visible: isVisible,name, price, type,} = product ?? {};
+    
+    const productType = type?.replace(/^\w/, (c: string) => c.toUpperCase());
+
+    if (isLoading || isInfoLoading) return <Loading />;
+    if (error) return <ErrorMessage error={error} />;
+
+  return (
+        <>
+        <Panel header="Basic Information">
+            <Text bold>Product name</Text>
```

Comment:
rather than using `Text bold`, we might actually make use of header here to denote that this is actually a header for the text below. Please be sure to import `H4` from BigDesign

```suggestion
import { H4, Panel, Text } from "@bigcommerce/big-design";
...
            <H4>Product name</H4>
```

Reply:
updated